### ISSUE TITLE
Test RSS source (part of credit mining test suite) is segfaulting

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#2795]
See the following stack trace:

```
test rss source ... /home/jenkins/workspace/GH_Tribler_PR_tests_linux/gumby/scripts/wrap_in_temp_home.sh: line 34: 35657 Segmentation fault      (core dumped) $*
+ FAILURE=139
+ rm -fR /home/jenkins/workspace/GH_Tribler_PR_tests_linux/tmp/tmp.c4f0JVnybU
+ exit 139
+ FAILURE=139
+ kill %Xvnc
+ exit 139
```

Unfortunately, I don't have any more info so this error might be *very* hard to debug. I've seen it once or twice before. @ardhipoetra do you have any idea what could be causing this?

Also, it only happend on Linux.

**Comment by ardhipoetra** [comment]
I can't recall anything that may cause this inside Tribler. What is the libtorrent version? Some of the function in specific libtorrent may cause this, CMIIW.

**Comment by devos50** [comment]
@ardhipoetra 

```
root@bbq:~# python
Python 2.7.11 (default, Dec  9 2015, 00:29:25) 
[GCC 5.3.1 20151205] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import libtorrent
>>> libtorrent.version
'1.0.7.0'
```

**Comment by devos50** [comment]
Fixed by #3390
